make_ghpages/make_pages.py

```diff
@@ -134,6 +148,33 @@ def get_html_provider_fname(provider_id):
     return "{}.html".format(simple_string)
 
 
+def validate_childdb(url: str) -> dict:
+    """ Run the optimade-python-tools validator on the child database.
+
+    Parameters:
+        url: the URL of the child database.
+
+    Returns:
+        dictionary representation of the validation results.
+
+    """
+    import dataclasses
+    from traceback import print_exc
+    validator = ImplementationValidator(
+        base_url=url,
+        run_optional_tests=False,
+        verbosity=-1,
```

Comment:
Wasn't the setting of `verbosity` to `-1` and internal manoeuvre within the class? The "proper" way would be to pass `json=True` or something, right? Or was this option only available for the invocation through the CLI?

Reply:
It is, I raised an issue about this in our repo that our verbosity is a bit wonky. I think perhaps changing this to 0 makes the most sense and then we can reflect that in our repo too. Even the JSON option prints out the summary, which I don't think is that useful here.

(When I added the JSON output to the validator, I didn't realise the dashboard was already written in Python and Jinja, I thought we were going to have to call the validator in the shell)

Reply:
```suggestion
        verbosity=0,
```